test_security/test/test_secure_publisher.cpp

```diff
@@ -77,8 +77,10 @@ int main(int argc, char ** argv)
   std::shared_ptr<rclcpp::node::Node> node = nullptr;
   try {
     node = rclcpp::Node::make_shared(node_name);
-  } catch (std::runtime_error &) {
-    fprintf(stderr, "should not have thrown!");
+  } catch (std::runtime_error & exc) {
+    std::cerr << "should not have thrown!\n" << exc.what();
```

Comment:
Use `printf` instead.

Same below.

Reply:
In the current test_communication, all c++ executables print to standard output using `std::cout` and error messages using `fprintf`. Which one is preferred ? I'm happy to update accordingly

Reply:
https://github.com/ros2/ros2/wiki/Developer-Guide#c-specific

Reply:
done in https://github.com/ros2/system_tests/pull/230/commits/a55fe0402dc60dd538c2d44bd6b246be2480f45c I'll open a follow-up PR for the tests in this repo but not part of the new `test_security` package